### PR TITLE
feat(client): ✨ Show users answer on prompt page when they click done

### LINKED ISSUE #156: Show confirmation that Catniss has submitted their answer.

### PR DESCRIPTION
**Pull request by Muaad2001**
When user hits done on prompt page their answer will be displayed, If they start typing another answer it won't update the answer until they hit done.

### CODE REVIEW

File: client/src/components/Prompt.svelte [hunk 1]
```diff
@@ -76,7 +82,10 @@
     <Textfield input$id="prompt-answer" bind:value={answer} bind:dirty invalid={dirty && error != undefined} required>
       <HelperText validationMsg slot="helper">{error ?? ""}</HelperText>
     </Textfield>
-    <Button type="submit" disabled={error != undefined}><Label>Done</Label></Button>
+    <Button type="submit" disabled={error != undefined} on:click={showAnswer}><Label>Done</Label></Button>
```
Comment (nstringham) [inline client/src/components/Prompt.svelte]: this should be in the submit event listner on the form

File: client/src/components/Prompt.svelte [hunk 2]
```diff
@@ -76,7 +82,10 @@
...
+    {#if showUserAnswer}
```
Comment (nstringham) [inline client/src/components/Prompt.svelte]: why not just do `displayAnswer != ""` or even make displayAnswer start as undefined and display if it is not undefined

**Review by nstringham — CHANGES_REQUESTED** [review]
I think this would be better if it was doing database reads like we do for voting but I guess it's ok like this

**Review by zaverbac — APPROVED** [review]

**Comment by SBAsai** [comment]
I think that when the timer hits 0:00, you shouldn't be able to keep entering in answers and have `Your Answer:` pop up when the answer isn't saved

**Comment by nstringham** [comment]
> I think that when the timer hits 0:00, you shouldn't be able to keep entering in answers and have `Your Answer:` pop up when the answer isn't saved

I like this idea but it should probably be another card

I created #166 which would resolve this

**Review by nstringham — APPROVED** [review]

**Comment by nstringham** [comment]
Requesting @gchamb's review because I think he said he wanted this to be a database subscription

**Comment by nstringham** [comment]
merging this and moving the firebase reads to #167

**Pull request merged**